game/src/core/components/DrawComponent.java

```diff
@@ -340,148 +323,68 @@ public void animationMap(final Map<String, Animation> animationMap) {
     this.animationMap = new HashMap<>(animationMap);
   }
 
-  /**
-   * Loading animation assets.
-   *
-   * <p>Checks if the game is running in a JAR or not and will execute the corresponding loading
-   * logic.
-   *
-   * <p>TODO: This is only a quick fix! We need to implement a proper way of loading assets. (See <a
-   * href="https://github.com/Dungeon-CampusMinden/Dungeon/issues/1361">Issue #1361</a>)
-   */
-  private void loadAnimationAssets(final IPath path) throws IOException {
-
-    Thread thread = Thread.currentThread();
-    StackTraceElement[] stack = thread.getStackTrace();
-
-    StackTraceElement currentElement = null;
-    Class<?> clazz = null;
-    for (int i = 1; i < stack.length; i++) {
-      currentElement = stack[i];
-      if (!currentElement.getClassName().equals(DrawComponent.class.getName())) {
-        try {
-          clazz = ClassLoader.getSystemClassLoader().loadClass(currentElement.getClassName());
-        } catch (ClassNotFoundException e) {
-          System.err.println(
-              "Could not load class " + currentElement.getClassName() + " from stacktrace.");
-        }
-        break;
+  // See https://stackoverflow.com/a/28985785
+  public static List<URL> getResources(final String path) throws IOException {
```

Comment:
Das ist (btw) ein "Trick", denn anscheinend kann man nur mit `getResourceAsStream` alle Resources an der richtigen Stelle durchsuchen. (eine genauere Beschreibung ist dazu in dem StackOverflow-Thread)

Aber dadurch entfällt auch die Fallunterscheidung, wo die Resources genau sind.